spk/jackett/Makefile

```diff
@@ -1,33 +1,38 @@
 SPK_NAME = jackett
-SPK_VERS = 0.7.1483
-SPK_REV = 4
+SPK_VERS = 0.8.490
+SPK_REV = 5
 SPK_ICON = src/jackett.png
 DSM_UI_DIR = app
 
 DEPENDS = cross/busybox cross/curl cross/$(SPK_NAME)
-SPK_DEPENDS = "mono>=4.2.1.102-6"
+# depend on mono 5.4 for BTLS HttpWebClient support
+SPK_DEPENDS = "mono>=5.4"
 
-MAINTAINER = SynoCommunity
-DESCRIPTION = "Jackett works as a proxy server: it translates queries from apps into tracker-site-specific http queries, parses the html response, then sends results back to the requesting software. Jackett is a single repository of maintained indexer scraping & translation logic - removing the burden from other apps."
-ADMIN_PORT = 9117
+MAINTAINER = 'kaso17 (Jackett), m4tt075 (spksrc integration)'
+DESCRIPTION = "Jackett works as a proxy server. It translates queries from apps into tracker-site-specific http queries, parses the html response, then sends results back to the requesting software. Jackett is a single repository of maintained indexer scraping and translation logic - removing the burden from other apps."
 RELOAD_UI = yes
 DISPLAY_NAME = Jackett
 BETA = 1
-CHANGELOG = Updated to version 0.7.1483
-
+STARTABLE = yes
+CHANGELOG = "1. Update Jackett to version 0.8.490, 2. Use DSM 5+6 generic service"
 HOMEPAGE = https://github.com/Jackett/Jackett
-LICENSE  = GPLv2
+LICENSE = GPLv2
+
+# 'auto' reserved value grabs SPK_NAME
+SERVICE_USER = auto
+SERVICE_SETUP = src/service-setup.sh
 
-INSTALLER_SCRIPT = src/installer.sh
-SSS_SCRIPT       = src/dsm-control.sh
-FWPORTS          = src/${SPK_NAME}.sc
-CONF_DIR         = src/conf/
+# Service configuration
+SERVICE_PORT = 9117
+SERVICE_PORT_TITLE = Jackett(HTTP)
 
-INSTALL_PREFIX = /usr/local/$(SPK_NAME)
+# Admin link
+ADMIN_PORT = $(SERVICE_PORT)
```

Comment:
Is it relevant to consider SERVICE_PORT as ADMIN_PORT ? Probably not, but I prefer to keep it for "backward compatibility"

Reply:
Sorry, not sure I understand. Do you want me to remove the comment and the ADMIN_PORT definition or do you want me to keep it?

Reply:
It depends if link generated in package view is really usable (often `http://ip:ADMIN_PORT/`) whereas `SERVICE_PORT` is used to provide application link in `app/config` which allows finer settings like `https` and context path in url.
So if "admin link" in package view is not usable as-is, please remove it. If it works, let it there.

Reply:
It is usable and works correctly, so I kept it.